infrastructure/validators/build-a-community.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Build a Community
+description: Learn how to build a validator community, attract nominations, and establish trust with nominators through best practices, transparency, and engagement.
+---
+
+# Build a Community
+
+## Introduction
+
+Building a solid validator community is crucial for attracting nominations and gaining trust in the Polkadot ecosystem. Validators must go beyond simply running nodes—they should engage with nominators, establish transparency, and build a reputation for reliability and security. This guide offers strategies for setting up an on-chain identity, maintaining transparency, and participating actively in governance and the broader community. By following these steps, validators can gain long-term nominations and build trust.
+
+## Attract Nominations
+
+Setting up a validator is just the beginning. Attracting nominations requires effort and visibility within the community. Nominators need to trust a validator before staking with them, and validators must stand out by providing reliable services and building a recognizable brand.
+
+Successful validators focus on the following:
+
+- **Operational Excellence** - running reliable nodes and avoiding slashing incidents
+- **Community Engagement** - participating in ecosystem discussions, governance, and validator forums
+- **Reputation** - creating a clear, trusted presence through transparency and consistent communication
+
+Both nominators and validators are at risk if slashing occurs, so building trust and showing commitment is vital. Validators should consistently communicate their reliability and transparency to ensure nominators feel secure when staking.
+
+### Gain Visibility
+
+Validators must be easily recognizable to attract nominations. Validators with clear, visible identities are more likely to gain trust from nominators than those with anonymous addresses.
+
+Steps to increase visibility:
+
+- **Set an On-Chain Identity** - ensure your validator has an on-chain identity that nominator dashboards can display. Validators with visible identities are easier for nominators to trust
```

Comment:
Link for set an identity guide: https://wiki.polkadot.network/docs/learn-guides-identity